### ISSUE TITLE
Add parameters table1_name,table2_name in data_compare method.

### ISSUE THREAD

**Issue by anilkumarpanda** [SauceCat/pydqc#23]
Current implementation of data_compare assigns default values of 'table1' and 'table2' to the dataframes.

![image](https://user-images.githubusercontent.com/7496719/79071259-1cbc3580-7cdb-11ea-80dc-be56cef7c472.png)

However, it would be good if the user can specify name of the dataframe, to be able to distinguish the dataframes better.

The signature can be as follows:
distribution_compare_pretty(_df1, _df2, _df1_name='table1',_df2_name='table2', col, figsize=None, date_flag=False)

**Comment by anilkumarpanda** [comment]
I would be happy to make the changes, if you agree.

**Comment by sosuneko** [comment]
Yes, please help create a PR and everyone can use the same function.

2020年4月12日(日) 午後10:40 Anilkumar Panda <notifications@github.com>:
